ports/serioussam-tfe/Serious Sam - The First Encounter.sh

```diff
@@ -41,18 +45,19 @@ else
   source "${controlfolder}/libgl_default.txt"
 fi
 
-if [ "$LIBGL_FB" != "" ]; then
-export SDL_VIDEO_GL_DRIVER="$GAMEDIR/gl4es/libGL.so.1"
-export LIBGL_ES=1
-export LIBGL_GL=14
-fi 
-
+export LD_LIBRARY_PATH="$GAMEDIR/Bin/libs:/usr/lib32:$LD_LIBRARY_PATH"
 export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
 
-$GPTOKEYB "ssam-tfe" -c "$GAMEDIR/serioussam.gptk" &
+export SDL_VIDEO_GL_DRIVER="$GAMEDIR/gl4es/libGL.so.1"
+#if [ "$LIBGL_FB" != "" ] && [ "$CFW_NAME" != 'ROCKNIX' ]; then
```

Comment:
You're overwriting the Rocknix behaviour from https://github.com/PortsMaster/PortMaster-GUI/blob/main/PortMaster/libgl_ROCKNIX.txt

Mind that Rocknix also has a Libmali switch which then requires the exports.